Sources/Puree/Output/BufferedOutput.swift

```diff
@@ -86,11 +91,24 @@ open class BufferedOutput: InstantiatableOutput {
 
     public func emit(log: LogEntry) {
         readWriteQueue.sync {
+            if let logSizeLimit = configuration.chunkDataSizeLimit, (log.userData?.count ?? 0) > logSizeLimit {
+                // Data whose size is larger than limit will never be sent.
+                return
+            }
+
             buffer.insert(log)
             logStore.add(log, for: storageGroup, completion: nil)
 
             if buffer.count >= logLimit {
                 flush()
+            } else if let logSizeLimit = configuration.chunkDataSizeLimit {
```

Comment:
How about executing `flush` before adding a new log?

Reply:
It seems to behave the same way